bump_definitions.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Axis Communications AB.
+# For a full list of individual contributors, please see the commit history.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Creates new versions of existing type definitions by increasing the
```

Comment:
https://github.com/eiffel-community/eiffel/issues/387 says 

> It should also add an entry to the history table.

Was that skipped due to complexity or forgotten?

Reply:
Oops, just forgotten. Added a `--history-entry` flag for this.